src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -240,6 +242,7 @@ public void FailFast_ExceptionStackTrace_StackOverflowException()
 
         [Trait(XunitConstants.Category, XunitConstants.IgnoreForCI)] // fail fast crashes the process
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework does not have dotnet/coreclr#16622")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/29869", TargetFrameworkMonikers.Uap)]
```

Comment:
Why add the github links to the ActiveIssue attribute here rather than just the issue number?

Reply:
There was a discussion awhile back and some people requested the URL so the could click on it..

Reply:
> There was a discussion awhile back and some people requested the URL so the could click on it..

Is that our new policy then?

I quick search reveals 1446 ActiveIssue attributes in the codebase.  Of those, only 149 use strings instead of numbers.  Of those, only 50 are urls to the issues in corefx.  So, very much in the minority.

I don't have a strong opinion about which approach we take.  But I do appreciate consistency, and I'm not sure why we'd actively choose to add more in the url form unless we plan to convert all of them as such.

Reply:
>Is that our new policy then?

I don't remember that.  I remember it was using the number.